src/Generators/Microsoft.Gen.MetadataExtractor/MetadataEmitter.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis;
+using Microsoft.Gen.ComplianceReports;
+using Microsoft.Gen.MetricsReports;
+using Microsoft.Gen.Shared;
+
+namespace Microsoft.Gen.MetadataExtractor;
+internal sealed class MetadataEmitter : EmitterBase
+{
+    private readonly MetricDefinitionEmitter _metricDefinitionEmitter;
+    private readonly ComplianceReportEmitter _complianceReportEmitter;
+    private readonly string _rootNamespace;
+    private readonly int _indentationLevel = 2;
+    public MetadataEmitter(string rootNamespace)
```

Comment:
Missing blank line after fields
```suggestion

    public MetadataEmitter(string rootNamespace)
```